docs/content/en/docs/soarca-extensions/fin-python-library.md

```diff
@@ -0,0 +1,210 @@
+---
+title:  Fin Python Library
+description: >
+    Documentation of the Python Fin library
+categories: [extensions, architecture]
+tags: [fin, python]
+weight: 2
+date: 2024-04-10
+---
+
+For documentation about the fin protocol we refer the reader to de documention page of [SOARCA](https://cossas.github.io/SOARCA/docs/soarca-extensions/fin-protocol/)
```

Comment:
@xncz8h please fix typo: the reader to `the` documentation page. Plus change link name SOARCA to `Fin Protocol`